app/utils.py

```diff
@@ -108,3 +108,11 @@ def setup_logger(name, log_file, level=logging.INFO):
     logger.addHandler(handler)
 
     return logger
+
+
+def reindex_search(query):
+    print("Reindexing Database")
+
+    index.clear_objects()
+    db_list = [u.serialize_algolia_search for u in query.all()]
+    index.save_objects(db_list)
```

Comment:
This is the code to rebuild the search index from our database. I haven't implemented it yet because we haven't decided where/how we want this ran. It really shouldn't be ran unless you're starting up and haven't created an index before or the data changed when the server was down.

Reply:
Is this supposed to do the same thing as the code you added to `run.py`? If not, what are the differences?

Reply:
I've removed this.